### PR TITLE
[Synfig Studio] Eyedropper tool: Added ability to select fill color using Ctrl+click

### PR DESCRIPTION
**Pull request by luboshnikov**
#1840

### CODE REVIEW

File: synfig-studio/src/gui/states/state_eyedrop.cpp
Comment (ice0) [inline synfig-studio/src/gui/states/state_eyedrop.cpp:148]: Ok, looks good, but 2 notices:
1) We are using tabs for whole project, so please use tabs here for indentation;
2) I think code can be optimized more. For example:
```
Color color(canvas_view->get_canvas()->get_context(canvas_view->get_context_params()).get_color(event.pos));

if ((event.modifier & GDK_CONTROL_MASK) == GDK_CONTROL_MASK) {
	synfigapp::Main::set_fill_color(color);
} else {
	synfigapp::Main::set_outline_color(color);
}
studio::App::dialog_color->set_color(color);

return Smach::RESULT_ACCEPT;
```
in this way you can clearly see which part is really changing.

Comment (luboshnikov) [reply synfig-studio/src/gui/states/state_eyedrop.cpp]: So I have fixed.

**Review by ice0 — APPROVED** [review]

**Comment by ice0** [comment]
Please squash commits to one or enable checkbox "Allow edits from maintainers", so I can squash it myself before merging.

https://docs.github.com/assets/images/help/pull_requests/allow-maintainers-to-make-edits-sidebar-checkbox.png

**Comment by luboshnikov** [comment]
I enabled the "Allow editing from maintainers" checkbox.

**Comment by ice0** [comment]
Merged. Thank you!

**Pull request merged**
